What are Bots?

jerryheir opened the pull request:
I didnt know where to put the link, but I think this is good like this

jerryheir committed:
What are Bots?

I didnt know where to put the link, but I think this is good like this

0sc commented on what-are-bots:
```diff
@@ -0,0 +1,17 @@
+WHAT ARE BOTS?
+
+Bots are also known as Internet Robots, and just as the name Robot implies; Bots make things easier. They can handle repetitive tasks and much more. Formerly, the popular version of bots came in the form of malware, but that has begun to change with tech companies like Facebook, Telegram, Slack providing a more useful version of this technology, and they represent CHAT BOTS.
```
Could you add a link for this 'Formerly, the popular version of bots came in the form of malware'

0sc commented on what-are-bots:
```diff
@@ -0,0 +1,17 @@
+
+THIS IS WHY CHATBOTS ARE IDEAL FOR YOUR IDEA AND BUSINESS.
+
+So let's be honest, compare how many apps are in the app stores to how many people actually use, and why is that? It's simply because no one wants to have to DOWNLOAD and INSTALL. Messenger ChatBots solve this problem, where users get to interact with your bots without having to download or install anything. They are the new apps with a currently emerging market, where big tech companies are curr ...
```
Hmm ... this sentences `So let's be honest, compare how many apps are in the app stores to how many people actually use, and why is that?` seems incomplete. Could review it.

0sc replied:
I see the idea behind this last paragraph which is to make a case for why readers should embrace bot technologies. Do you think it would be better if you expounded more on these reasons?

Also reading from the top down, it felt a bit disconnected from the story. If the story has a summary or recap of some sort, it could lead fluidly into this section. Something like saying: (after the story)

 To recap:
* Websites came on board the internet to provide access location to different information
* With the boom of mobile devices, mobile apps grew to provide easy access to various experiences to users enhancing the overall internet connectivity.
* To help solve the problems of overwhelming mobile apps and to boost business (by easy customer acquisition) some big companies opened up their platform (e.g messenger, telegram, slack) to allow businesses build custom/automated experiences on them.

> You can then follow up with

Leveraging this bot platform is a great to avoid the growing [mobile app fatigue](https://techcrunch.com/2016/02/03/app-fatigue/). Of the many apps in the app stores, only a [few have user actively using them](http://fortune.com/2016/08/16/app-fatigue-is-taking-a-toll-on-smartphone-owners/). Why? It's simply because no one wants to have to DOWNLOAD and INSTALL. Chatbots solve this ... 

------

This is just an example. Feel free to modify the approach.

0sc reviewed (changes requested):
Nice, @jerryheir it almost good to go.

I left some comments bordering mostly on:
* Adding links to backup some assertion. While making sure these resources are easy to understand, it's also important to ensure, they have accurate information. For example, this Wikipedia article, https://en.wikipedia.org/wiki/Chatbot, has some insights into the history of chatbots but neither does it highlight malicious chatbots nor does it accord chatbot popularity to them.

* Summarizing the story afterwards to highlight the message being conveyed. It's easy to get lost enjoying the story will missing out on the salient points. A summary that presents this points afterward will be very valuable.

* [Markdown](http://www.markdowntutorial.com/)?[Here's a quick overview to get you started]. Check out this two posts, https://github.com/jerryheir/Messenger_bot/blob/aacbc026bbbf98ae155ea648ea0092c9df1e6c1c/what-are-bots and https://gist.github.com/andela-ooranagwa/ee052c4bdefe644c428557adcfaaa61c to see the difference with and without markdown formatting. I see the markdown formatting got lost somehow but no worries, I can help with it once you're good to go.

jerryheir committed:
Updated

I have used all your awesome suggestions... Please tell me if I have to make more corrections

jerryheir committed:
Update what-are-bots.md

jerryheir committed:
Update what-are-bots.md

0sc commented on what-are-bots.md:
```diff
@@ -0,0 +1,20 @@
+### WHAT ARE BOTS?
```
Since this is the heading of the article. One `#` would be more appropriate. i.e.

# WHAT ARE BOTS?

And for subsequent headings, you can use two `#`. ie.
## A subheading

0sc commented on what-are-bots.md:
```diff
@@ -0,0 +1,20 @@
+
+One day, the King took a trip and when he finally got back, he noticed a new disorganization problem, this time there were so many territories, and mansions scattered all over the place, relevant and irrelevant. He was not happy with this. And so he called a meeting and stated this problem saying "_Why is every Knight building a mansion or territory for every single purpose when he can just have  ...
+
+##### THIS IS WHY CHATBOTS ARE IDEAL FOR YOUR IDEA AND BUSINESS.
```
What do you think of replacing this with something that links the section below to the story above? Something like:
To summarize;
* website ...

0sc commented on what-are-bots.md:
```diff
@@ -0,0 +1,20 @@
+* With the boom of mobile devices, mobile apps grew to provide easy access to various experiences to users enhancing the overall internet connectivity.
+* To help solve the problems of overwhelming mobile apps and to boost business (by easy customer acquisition) some big companies opened up their platform (e.g messenger, telegram, slack) to allow businesses build custom/automated experiences on them.
+
+Leveraging this bot platform is a great to avoid the growing mobile app fatigue. Of the many apps in the app stores, with only a few have users actively using them. Why? It's simply because no one wants to have to DOWNLOAD and INSTALL. Chatbots solve this ...
```
What do you think of updating this section with:

Leveraging this bot platform is a great to avoid the growing [mobile app fatigue](https://techcrunch.com/2016/02/03/app-fatigue/). Of the many apps in the app stores, only a [few have user actively using them](http://fortune.com/2016/08/16/app-fatigue-is-taking-a-toll-on-smartphone-owners/). Why? It's simply because no one wants to have to DOWNLOAD and INSTALL. *Chatbots solve this.*

----
The main difference is the added links

0sc reviewed (changes requested):
Good to go 💯 x 💯 x 🚤.

Just fix the formatting and let's merge it in.

jerryheir committed:
Update what-are-bots.md

jerryheir committed:
Done ...
